docs/guides/references/migration-guide.mdx

```diff
@@ -6,7 +6,66 @@ title: Migration Guide
 
 This guide details the changes and how to change your code to migrate to Cypress
 version 13.0.
-[See the full changelog for version 13.0](/guides/references/changelog#13-0-0).
+[See the full changelog for version 13.0](https://docs.cypress.io/guides/overview/why-cypress).
+
+### Cypress Cloud Test Replay
+
+[Test Replay](https://docs.cypress.io/guides/overview/why-cypress) is enabled by default in `v13.0` of the Cypress App.
+
+You may need to whitelist `capture.cypress.io` if you work with a strict VPN. See our FAQ section about [VPN subdomain whitelisting](/faq/questions/cloud-faq#Im-working-with-a-restrictive-VPN-Which-subdomains-do-I-have-to-allow-on-my-VPN-for-Cypress-Cloud-to-work-properly).
+
+You can [opt out](https://docs.cypress.io/guides/overview/why-cypress) of this feature in Cloud project-level settings.
+
+<!---
+todo: replace urls for merge to v13 branch:
+
+changelog- /guides/references/changelog#13-0-0
+test replay- /guides/cloud/debugging/test-replay
+opt out- /guides/cloud/debugging/test-replay#Opt-out-of-Test-Replay
+--->
+
+### Video updates
+
+#### `video` is set to false by default
+
+Whether Cypress will capture a video of the tests run with `cypress run`.
```

Comment:
I don't understand the point of this line.
```suggestion
```

Reply:
@jennifer-shehane Yeah, I used the first line under these headings to provide context for what the particular config is. I suppose you may know already if you're here. It does seem a little out of place without more surrounding it. Do you see value in that if it's made more clear in the verbiage or just better without?